Makefile

```diff
@@ -29,20 +29,15 @@ $(MAINBOARDS):
 	cd $(dir $@) && make
 
 firsttime:
-	rustup override set $(TOOLCHAIN_VER)
-	rustup component add rust-src llvm-tools-preview rustfmt clippy
-	rustup target add riscv64imac-unknown-none-elf
-	rustup target add riscv32imc-unknown-none-elf
-	rustup target add armv7r-none-eabi
-	rustup target add aarch64-unknown-none-softfloat
+	# --default-toolchain is purely an optimization to avoid downloading stable Rust first.
+	curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain $(TOOLCHAIN_VER)
```

Comment:
I'd put this curl-pipe-sh in `debiansysprepare` to avoid running it for people who would already have rustup installed through other means, since it's available as a package usually (e.g. in Arch) and redundant copies may cause issues.